documentation/getting_started/mqtt-client-openshift.adoc

```diff
@@ -1,5 +1,5 @@
-[[mqtt]]
-==== Connecting using MQTT
+[[mqtt-{context}]]
```

Comment:
[id=mqtt-{context}]

Reply:
I've just removed this reference for now, as i think it would require a larger restructure of the docs due to this being used in multiple getting started contexts (and thus failing the build).